email/emailer.go

```diff
@@ -37,45 +24,39 @@ type EmailConfig struct {
 
 type EmailSender struct {
 	Conf EmailConfig
-	Send func(string, smtp.Auth, string, []string, []byte) error
 }
 
 func InitEmailer() {
-	Emailer = NewEmailSender(EmailConfig{
-		ServerHostPort: env.MustGetString("SMTP_HOST"),
-		Username:       env.MustGetString("SMTP_USER"),
-		Password:       env.MustGetString("SMTP_PASSWORD"),
-	})
-}
-
-func NewEmailSender(conf EmailConfig) Interface {
-	return &EmailSender{conf, smtp.SendMail}
-}
-
-// email.Interface is an interface to a struct that 'sends' emails. It allows the send function to be mocked out for testing.
-type Interface interface {
-	SendMail(email *Email) error
+	Emailer = EmailSender{
+		Conf: EmailConfig{
+			ServerHostPort: env.MustGetString("SMTP_HOST"),
+			Username:       env.MustGetString("SMTP_USER"),
+			Password:       env.MustGetString("SMTP_PASSWORD"),
+		},
+	}
 }
 
 // Send sends a simple email via a smtp gateway using TLS
-func (e *EmailSender) SendMail(email *Email) error {
-	host, _, _ := net.SplitHostPort(e.Conf.ServerHostPort)
-	auth := smtp.PlainAuth("", e.Conf.Username, e.Conf.Password, host)
+func Send(content *Email) error {
+	once.Do(func() {
```

Comment:
Currently InitEmailer() gets called by services that use this package.

Maybe you should do the once.Do() inside InitEmailer()?